packages/protocol/contracts/test/mocks/BadTokenMock.sol

```diff
@@ -1,11 +1,13 @@
+// Note: perhaps we should remove the comments / etc. that are not relevant (e.g. first blood, oz) since this is just a broken mock (see the aragonOS mocks)
 // Copied from https://github.com/OpenZeppelin/openzeppelin-solidity/blob/a9f910d34f0ab33a1ae5e714f69f9596a02b4d91/contracts/token/ERC20/StandardToken.sol
 
+// Perhaps we should use the same bad token mocks as aragonOS?
```

Comment:
We cannot import them, as we are using solidity 5 here, so being just a mock with a very limited purpose I don’t think it’s worth copying them and porting them.